### DISCUSSION TITLE
Is there any possibility to load a scene as source into the ModelViewer?

### DISCUSSION THREAD

**Discussion by zntr1** [google/model-viewer#4485] [Q&A]
Hello,

I am currently trying something out and I wonder, whether it is possible to load a THREE.js Scene directly into the ModelViewer. 

Any Ideas?

Thanks guys

**Comment by zntr1** [comment]
Anyone has an idea? Is my question not clear enough? Just in case, I ask my question again with a different wording :).

So ModelViewer uses Three.js as an underlying library. ThreeJS uses scenes that are later on being rendered. As far as I can see, ModelViewer also uses scenes and renders them, so a model is visible to the user in the end. Now my question is: Is there any method to actually load a scene directly to the model-viewer element instead of having to load a file with the src-attribute?

**Reply by elalish** [reply]
Not as a public API, as there are many gotchas to get this to work. However, if you simply get our internal `$scene` symbol, you can easily hack anything you want. There's even a `setScene` method on it, but keep in mind you're on your own for figuring out how to make it work - this is unsupported.
